add zip configuration for dicom zips

kalebdfischer opened the pull request:
The changes in this PR are needed for DICOM zip archive processing.

kalebdfischer committed:
add zip configuration

kalebdfischer committed:
fix flipped output and input-regex

cookpa commented:
Hi Kaleb, 

Will this prevent fw import dicom from working correctly on files without a recognized extension? Not all our raw data ends with .dcm or similar.

kalebdfischer commented:
> Hi Kaleb,
> 
> Will this prevent fw import dicom from working correctly on files without a recognized extension? Not all our raw data ends with .dcm or similar.

Ah, this file is being used for the reaper and CLI, isn't it? These services don't have the zip archive code, so I'll push a commit with a new file.

kalebdfischer committed:
Revert "fix flipped output and input-regex"

This reverts commit f1f7c81064a47cbf27001d155f9faa27b3d2050c.

kalebdfischer committed:
Revert "add zip configuration"

This reverts commit bb0355023dbaa3aad0e41b34de17a45f5041c8c7.

kalebdfischer committed:
add file for PHI cleanup deployment

this file accommodates de-identification of existing dicom zip archives with the migration toolkit (not yet available in CLI).

cookpa commented:
Thanks. Is this new file for use in the projects on the Flywheel site?
